cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,44 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+
+`Release 4.3.2 <http://docs.cask.co/cdap/4.3.2/index.html>`__
+=============================================================
+New Features
+------------
+
+- :cask-issue:`CDAP-12771` - Added GCS connection to Data Prep.
```

Comment:
shouldn't there be an issue about S3 too?

Reply:
yes, added it